tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -156,6 +213,19 @@ internal static bool CheckRegistry(IRegistryService? registry = null)
             }
         }
 
+        private static bool IsExpectedProfilerFile(string fullPath)
+        {
+            var fileName = Path.GetFileName(fullPath);
+
+            if (RuntimeInformation.IsOSPlatform(System.Runtime.InteropServices.OSPlatform.Windows))
+            {
+                return "Datadog.Trace.ClrProfiler.Native.dll".Equals(fileName, StringComparison.OrdinalIgnoreCase);
+            }
+
+            // Paths are case-sensitive on Linux
+            return "Datadog.Trace.ClrProfiler.Native.so".Equals(fileName, StringComparison.Ordinal);
```

Comment:
This won't work for Macs :P

Reply:
Yeah, we'll cross that bridge when we have customers trying to make the tracer work on a mac 😛

Reply:
:oldmanyelling: